### PR TITLE
Update CI test pipeline

### LINKED ISSUE #110: Update test pipeline
Use tarantool-java test pipeline. 
Add jacoco coverage reports.

### PR DESCRIPTION
**Pull request by iDneprov**
- Add integration profile
- Run ce and ee tests in parallel
- Add jacoco coverage report

I haven't forgotten about:
- [x] Tests
- [ ] Changelog
- [ ] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:

Closes #110

### CODE REVIEW

**Review by ArtDu — CHANGES_REQUESTED** [review]

**Comment by ArtDu** [comment]
Make commit message(title and body), PR title and PR description more describeful

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+      - name: Jacoco Report to PR
+        id: jacoco
+        uses: madrapps/jacoco-report@v1.6.1
+        with:
+          paths: target/site/jacoco/jacoco.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-coverage-overall: 40
+          min-coverage-changed-files: 60
+          update-comment: true
```
[the remark this answers is not in the document]
Comment (iDneprov) [reply .github/workflows/tests.yml:116]: > May you add badge to README in this PR?
Lets do it in another task. 

> And do you accept this baselines?

[the remark this answers is not in the document]
Comment (ArtDu) [reply .github/workflows/tests.yml:116]: > Lets do it in another task.

Alright, create a ticket please

> This is numbers form tarantool-java. If you want to change them, lets discuss.

No needed

File: pom.xml [hunk 2]
```diff
@@ -203,10 +203,74 @@
...
+                            </includes>
+                            <excludes>
+                                <exclude>**/*EnterpriseIT.java</exclude>
+                            </excludes>
+                            <systemPropertyVariables>
+                                <logback.configurationFile>${logging.config}</logback.configurationFile>
+                                <logLevel>${logging.logLevel}</logLevel>
+                            </systemPropertyVariables>
+                            <trimStackTrace>false</trimStackTrace>
+                            <argLine>@{argLine} -Xmx1024m -XX:MaxPermSize=256m</argLine>
```
Comment (ArtDu) [inline pom.xml:268]: I think we can remove it 

image

File: pom.xml [hunk 3]
```diff
@@ -218,6 +282,7 @@
                         <configuration>
                             <includes>
                                 <include>**/*TestEnterprise.java</include>
+                                <include>**/*EnterpriseIT.java</include>
```
Comment (ArtDu) [inline pom.xml:285]: I don't see changes we have discussed 

You're right that we can use IT without Test and that's ok for idea regex.
But I'd like to have unification about this question. To have the same things in our every product

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsIT.java [hunk 4]
```diff
@@ -23,7 +23,7 @@
  * @author Alexey Kuzin
  */
 @Testcontainers
-public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest {
+public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsIT {
```
Comment (ArtDu) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsIT.java:26]: I don't see changes we have discussed

You're right that we can use IT without Test and that's ok for idea regex.
But I'd like to have unification about this question. To have the same things in our every product

**Review by ArtDu — CHANGES_REQUESTED** [review]
nit: I haven't seen commit message changings

File: pom.xml [hunk 5]
```diff
@@ -218,6 +282,7 @@
                         <configuration>
                             <includes>
                                 <include>**/*TestEnterprise.java</include>
+                                <include>**/*ITEnterprise.java</include>
```
[the remark this answers is not in the document]
Comment (ArtDu) [reply pom.xml]: It was my mistake that we can use IT without test word because IT is already Integration Test

**Review by ArtDu — APPROVED** [review]

**Pull request merged**